src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java

```diff
@@ -58,19 +63,15 @@ void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled(SoftAssertions so
 
     @SuppressWarnings("checkstyle:magicnumber")
     private void assertGpOutboundQueueMessages(SoftAssertions softly) throws IOException, JMSException, JSONException {
-        final List<Message> gpOutboundQueueMessages = IntStream.range(0, 6)
+        final List<Message> gpOutboundQueueMessages = IntStream.range(0, 2)
             .mapToObj(x -> getGpOutboundQueueMessage())
             .collect(Collectors.toList());
 
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(0));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(1));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(2));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(3));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(4));
-        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(5));
+        assertGpOutboundQueueMessages(softly, gpOutboundQueueMessages.get(0), fhirMessage1);
```

Comment:
Now we're testing multiple 'different' messages, i've removed 4 messages until we have more mappers in place.